test/mlir/onnx/onnx_canonicalization.mlir

```diff
@@ -222,3 +216,49 @@ func @test_transpose_fusion_removal(%arg0: tensor<10x11x12x13xf32>) -> tensor<10
   // CHECK-NEXT: return %arg0 : tensor<10x11x12x13xf32>
   "std.return"(%1) : (tensor<10x11x12x13xf32>) -> ()
 }
+
+// -----
+
```

Comment:
Please add one comment for each new test to state what they do. LLVM requires it, and I think its a good practice.